build.sbt

```diff
@@ -24,4 +24,6 @@ lazy val microservice = Project(appName, file("."))
     ScoverageKeys.coverageMinimumStmtTotal := 90,
     ScoverageKeys.coverageFailOnMinimum := false,
     ScoverageKeys.coverageHighlighting := true)
+
 scalacOptions += "-Wconf:cat=unused-imports&src=routes/.*:s"
+scalacOptions += "-Wconf:cat=unused&src=routes/.*:s"
```

Comment:
Warning `private val defaultPrefix in class Routes is never used`.
DefaultPrefix is autogenerated by Play framework in the target folder so removing it would be undone every time we compile. Is worth keeping anyway in case it is helpful in future.

`scalacOptions += "-Wconf:cat=unused&src=routes/.*:s"` suppresses unused warnings from the routes folder.